### PR TITLE
Pagination and Client cache daemon

### PR DESCRIPTION
**Pull request by johanbrandhorst**
This PR contains the LLB used for the pagination and client cache daemon. All code has already been reviewed.

### CODE REVIEW

File: enos/modules/docker_boundary/boundary-config-rate-limit.hcl [hunk 1]
```diff
@@ -35,8 +35,8 @@ controller {
     resources = ["host"]
     actions = ["read"]
     per = "auth-token"
-    limit = 5
-    period = "5s"
+    limit = 3
+    period = "10s"
```
Comment (johanbrandhorst) [inline enos/modules/docker_boundary/boundary-config-rate-limit.hcl:39]: Do we need to revert this again? I believe it was changed by @moduli while we were having problems with the e2e tests. I'm going to try reverting it and see if the tests are still passing. CC @talanknight

Comment (johanbrandhorst) [reply enos/modules/docker_boundary/boundary-config-rate-limit.hcl:39]: https://github.com/hashicorp/boundary/pull/4183 was when it was introduced.

Comment (johanbrandhorst) [reply enos/modules/docker_boundary/boundary-config-rate-limit.hcl:39]: Opened https://github.com/hashicorp/boundary/pull/4221 to see if we can revert this.

Comment (johanbrandhorst) [reply enos/modules/docker_boundary/boundary-config-rate-limit.hcl:39]: Looks like we still need this judging by the test results.

**Comment by johanbrandhorst** [comment]
I've reviewed the changes and think this is good to merge. If anyone has any questions please let me know, otherwise we should aim to merge this tomorrow Friday the 5th.

**Review by jimlambrt — APPROVED** [review]

**Review by mikemountain — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Manually fast forward merged 🎉

**Pull request closed without merging**